src/inferenceql/query/relation.cljc

```diff
@@ -8,7 +8,9 @@
 ;; These are not sequence functions, so the primary argument comes first.
 
 (defn relation
-  "Produces a relation from a sequence of maps."
+  "Produces a relation from a sequence of maps.
+
+  Returns the same collection, but w/ name and attr metadata attached."
```

Comment:
I could go either way. I chose the docstring, since it more likely to be seen, as a counterweight to the invisibility of metadata. But OTOH, it's not a public-facing fn, exactly, so maybe that's less important. I'll move it to comments.

Reply:
I personally tend to document a lot of fns that aren't visible to users, and write them either for future-me, or my fellow library devs.

Reply:
Ahh, I see. Yeah, that's probably less of a communication, and more of a note to myself while learning my way around. I was originally expecting a type, not a map with metadata, so I jotted down my surprise...